doc/LocalVMSetup.md

```diff
@@ -16,9 +17,29 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
 ## 3. Install a fresh CentOS 7.8.2003
 Automatic install or manual install. Basic installation is OK. When you build CORTX from source in next steps, dependant packages will be resolved and installed.
 
-## 4. You may need to add EPEL repo.
+## 4. You may need to configure your network settings
+run a test to check if you are connected to the network 
+
+     sudo nmcli d
+   
+if you have a network connection then skip on to step 4, otherwise follow the below commands:
+
+     sudo nmtui
+ 
+ -> Edit a connection, select your network interface and choose "Automatically connect" option (by pressing the space key) and then select OK.
+ 
+    sudo reboot now
+
+## 5. Connect to your VM using Putty (Optional)
+Get the ip address of your VM sudo nmcli will provide you with the ip address. Copy the ip address to your PuTTY console and click connect.
+
+## 6. Install git
+
+    yum install git
```

Comment:
* This is not a MUST-HAVE. If people are trying to install CORTX from PRMs, they don't need "git".
* If sudo is needed in modified step 4, then sudo is needed here, too.